### PR TITLE
Adding PyVISA-sim

### PR DESCRIPTION
**Pull request by mtsolmn**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mtsolmn** [comment]
@MatthieuDartiailh saw the comment about adding `pyvisa-sim` to `conda-forge` in this issue: https://github.com/pyvisa/pyvisa-sim/issues/62 and here: https://github.com/conda-forge/pyvisa-py-feedstock/issues/22

I doubt I will be able to meaningfully help maintain the package, but I've added you to the maintainer list. Comment to accept and please add anyone else who you think can help maintain this. Thanks

**Comment by MatthieuDartiailh** [comment]
I agree to maintain the package once this PR is merged.

**Comment by mtsolmn** [comment]
@conda-forge-admin, please rerender

**Comment by mtsolmn** [comment]
@MatthieuDartiailh do you have any idea why the pytests are failing? If you don't know and don't think it's an issue, then I would remove the purest commands from the recipe and publish. Let me know your thoughts, thanks.

**Comment by MatthieuDartiailh** [comment]
It looks like some changes in pyvisa (notably handling of GPIB secondary address) broke some of the tests of pyvisa-sim. I will have to try to fix them at one point, but I have limited bandwidth for this project.

**Comment by mtsolmn** [comment]
@MatthieuDartiailh I totally get it. I think the repository is fine as is, and the package is working nominally, even if the built-in tests haven't been appropriately updated. I've gone ahead and removed the tests from the CI pipeline for publishing.

**Comment by mtsolmn** [comment]
@conda-forge/help-python ready to merge :)

File: recipes/pyvisa-sim/meta.yaml
Comment (xhochy) [inline recipes/pyvisa-sim/meta.yaml:39]: This doesn't seem to be used in the tests here.
```suggestion
```

File: recipes/pyvisa-sim/meta.yaml
Comment (xhochy) [inline recipes/pyvisa-sim/meta.yaml:20]: Better use `>=`. `>` is not so common and harder to comprehend.

**Review by xhochy — CHANGES_REQUESTED** [review]

**Review by xhochy — APPROVED** [review]

**Pull request merged**
